MSBuildLaunchPad/ParserFactory.cs

```diff
@@ -7,6 +7,8 @@ public static class ParserFactory
         public static IParser Parse(string fileName)
         {
             string extension = Path.GetExtension(fileName);
+            var n = extension.Length;

```

Comment:
We should use a more descriptive name for naming the variable. How about `extensionLenght`?